apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_lib/fetchEvaluationCached.ts

```diff
@@ -0,0 +1,10 @@
+import { cache } from 'react'
+
+import { EvaluationsRepository } from '@latitude-data/core/repositories'
+import { getCurrentUser } from '$/services/auth/getCurrentUser'
+
+export const fetchEvaluationCached = cache(async (id: number) => {
+  const { workspace } = await getCurrentUser()
+  const evaluationScope = new EvaluationsRepository(workspace.id)
+  return evaluationScope.find(id).then((r) => r.unwrap())
+})
```

Comment:
Now that evaluation is fetched in layout and page is a perfect use case for `cache`